build.sbt

```diff
@@ -16,25 +16,24 @@ lazy val scoverageSettings = {
 
 lazy val compileDeps = Seq(
   ws,
-  "uk.gov.hmrc"       %% "bootstrap-frontend-play-27" % "5.11.0",
-  "com.typesafe.play" %% "play-json-joda"             % "2.7.4",
-  "uk.gov.hmrc"       %% "govuk-template"             % "5.69.0-play-27",
-  "uk.gov.hmrc"       %% "play-ui"                    % "9.6.0-play-27",
-  "uk.gov.hmrc"       %% "play-partials"              % "8.2.0-play-27",
-  "uk.gov.hmrc"       %% "simple-reactivemongo"       % "8.0.0-play-26",
+  "uk.gov.hmrc"       %% "bootstrap-frontend-play-28" % "5.9.0",
+  "com.typesafe.play" %% "play-json-joda"             % "2.9.2",
+  "uk.gov.hmrc"       %% "govuk-template"             % "5.70.0-play-28",
+  "uk.gov.hmrc"       %% "play-ui"                    % "9.7.0-play-28",
+  "uk.gov.hmrc"       %% "play-partials"              % "8.2.0-play-28",
+  "uk.gov.hmrc"       %% "simple-reactivemongo"       % "8.0.0-play-28",
   "uk.gov.hmrc"       %% "agent-mtd-identifiers"      % "0.25.0-play-27",
```

Comment:
Do we need to uplift https://github.com/hmrc/agent-mtd-identifiers first?

Reply:
That is the most recent version that I could see. There is nothing explicitly for play 2.8

Reply:
Yea is it ours? https://github.com/hmrc/agent-mtd-identifiers/blob/master/build.sbt#L30 It looks like the work has been done, just that there isn't a release for it? @geoffreywatson